src/external/elasticsearch.ts

```diff
@@ -0,0 +1,23 @@
+import { Client } from '@elastic/elasticsearch';
+
+import config from '../config';
+
+let esClient: Client;
+
+export async function getClient(): Promise<Client> {
```

Comment:
I see  in other files you use arrow function expressions... is there any reason why this one is a function declaration?

Reply:
Should probably be more consistent. When I'm writing root level functions in a module I tend to prefer function keyword to arrow.